api/src/main/java/org/openmrs/module/reporting/report/renderer/ReportTemplateRenderer.java

```diff
@@ -222,17 +222,22 @@ public String getExpressionSuffix(ReportDesign design) {
 	 * @return the value for the report template replacement given the initial value
 	 */
 	public Object getReplacementValue(Object initialValue) {
+
 		Object replacementValue = "";
-		if (initialValue != null) { 
-			if (initialValue instanceof Cohort) {
-				replacementValue = new Integer(((Cohort) initialValue).size());
-			} 
-			else if (initialValue instanceof IndicatorResult) {
-				replacementValue = new Double(((IndicatorResult) initialValue).getValue().doubleValue());
-			}
-			else {
-				replacementValue = initialValue;
+		if (initialValue != null) {
+			 if (initialValue instanceof IndicatorResult) {
+				IndicatorResult ir = (IndicatorResult) initialValue;
+				if(ir.getValue() instanceof Integer)
+				{

```

Comment:
Code conventions put the brackets on the same line

Reply:
Code conventions have been followed in the latest commit